### PR TITLE
remove held_by in favour of locations

### PR DESCRIPTION
**Pull request by pgwillia**
https://github.com/ualbertalib/NEOSDiscovery/issues/152
screenshot from 2018-10-01 12-57-16

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -17,7 +17,6 @@ def load_lookup_tables
     @languages ||= YAML.load_file("#{Rails.root}/config/languages.yml")
     @statuses ||= YAML.load_file("#{Rails.root}/config/statuses.yml")
     @item_types ||= YAML.load_file("#{Rails.root}/config/item_types.yml")
-    @library_codes ||= YAML.load_file("#{Rails.root}/config/library_codes.yml")
```
Comment (weiweishi) [inline app/controllers/application_controller.rb]: So this file was used only for this one purpose? to supply library code for "copied owned/held_by"? Wow!

File: app/views/catalog/_index_default.html.erb [hunk 2]
```diff
@@ -20,14 +20,8 @@
...
         <dd class="blacklight-format">
-          <% if document["held_by_tesim"] %>
-            <% @held_by_locations = [] %>
-            <% document["held_by_tesim"].first.split.each do |held_by| %>
-              <% @ual = true if (@library_codes[held_by.to_i].include?("University of Alberta") or (@library_codes[held_by.to_i].include?("NEOS Free"))) %>
-              <% @held_by_locations << @library_codes[held_by.to_i] %>
-            <% end %>
-            <%= @held_by_locations.join(", ") %>
-          <% end %>
+          <%= document["location_tesim"].join(', ') %>
```
Comment (weiweishi) [inline app/views/catalog/_index_default.html.erb]: Seems that there were some exceptions like "University of Alberta" or "NEOS Free". Are those presented in the location_tesim? Do we need to treat them separately?

Comment (pgwillia) [reply app/views/catalog/_index_default.html.erb]: Thanks for the reminder.  I didn't follow up on moving the @ual shield.  Will move this to a helper and add some tests.

Comment (pgwillia) [reply app/views/catalog/_index_default.html.erb]: Fixed this!

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (pgwillia) [reply app/controllers/application_controller.rb]: As far as I can tell...  Doesn't appear anywhere else, and searching the history on github doesn't make it super clear how or why this was added.

**Pull request merged**